build.gradle

```diff
@@ -188,14 +188,19 @@ dependencyUpdates.revision = 'integration'
 dependencyUpdates.resolutionStrategy = {
     componentSelection { rules ->
         rules.all { ComponentSelection selection ->
-            if (selection.candidate.version ==~ /[0-9].*SNAPSHOT/) {
+            if ( selection.candidate.module != "junit-pioneer" && selection.candidate.module!="javax.inject"  && selection.candidate.version ==~ /[0-9].*SNAPSHOT/ ) {
```

Comment:
javax.inject -> javax.inject:javax.inject ?

Reply:
I initially had the same idea but my tests with junit pioneer showed that it didn't work. I then added a print statement and saw that the module is just the real module name
```
 componentSelection { rules ->
        rules.all { ComponentSelection selection ->
print selection.candidate.module 
```